client/scenes/stream_gui.cpp

```diff
@@ -571,6 +579,27 @@ void scenes::stream::gui_settings(float predicted_display_period)
 	ImGui::PopStyleVar();
 }
 
+void scenes::stream::gui_bitrate_settings(float predicted_display_period)
+{
+	ImGui::PushFont(nullptr, constants::gui::font_size_large);
+	ImGui::Text("%s", _S("Use the right thumbstick to change the bitrate"));
+	ImGui::Text("%s", _S("Press A to go back"));
+	ImGui::Text("%s", fmt::format(_F("Bitrate: {}Mbit/s"), bitrate / 1'000'000).c_str());
+	ImGui::PopFont();
+
+	// Maximum speed 1 rad/s
+	float delta = application::read_action_float(foveation_pitch).value_or(std::pair{0, 0}).second * 20'000'000.f * predicted_display_period;
```

Comment:
We should do some renaming:
1 rad/s -> 20Mb/s (or just say we limit the adjustment speed)
foveation_pitch -> settings_adjust